rotkehlchen/tests/db/test_db.py

```diff
@@ -1370,3 +1375,83 @@ def test_values_are_present_in_db(database, enum_class, table_name):
     for enum_class_entry in enum_class:
         r = cursor.execute(query, (enum_class_entry.value,))
         assert r.fetchone() == (1,)
+
+
+def test_known_locations(user_data_dir, rotkehlchen_api_server):
+    """
+    Test that locations imported in different places are correctly stored in database
+    """
```

Comment:
```suggestion
    """Test that locations imported in different places are correctly stored in database"""
```

If it fits in one line keep it in one